frontend/appflowy_flutter/lib/mobile/presentation/database/card/card_detail/mobile_card_detail_screen.dart

```diff
@@ -420,6 +420,7 @@ class _TitleSkin extends IEditableTextCellSkin {
             fontSize: 23,
             fontWeight: FontWeight.w500,
           ),
+      onChanged: (text) => bloc.add(TextCellEvent.updateText(text)),
```

Comment:
What do you think about using a debounce?

Reply:
Yes I agree. I put it it in the text cell bloc here

![image](https://github.com/AppFlowy-IO/AppFlowy/assets/71320345/035ff79f-ee0e-4542-b2a1-90f136b17507)

Reply:
Amazing